packages/mobile/components/ToastContainer.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="typescript">
+    import { Alert, Toast } from 'shared/components'
+    import { notifications } from '@auxiliary/notification'
+    import { fade } from 'svelte/transition'
+
+    $: toasts = $notifications.map((notification) => ({
+        type: notification.type,
+        alert: notification.alert,
+        message: notification.message,
+        subMessage: notification.subMessage,
+        progress: notification.progress,
+        actions: notification.actions.map((action, actionIndex) => ({
+            ...action,
+            onClick: () => action.callback(notification, actionIndex),
+        })),
+    }))
+</script>
+
+<toast-container class="fixed top-0 flex flex-col z-10 w-full p-5">
+    <ul class="space-y-2">
+        {#each toasts as toast}
+            <li in:fade={{ duration: 100 }} out:fade={{ duration: 100 }}>
```

Comment:
perhaps we should add `|local` to avoid issues when a parent component gets destroyed
```suggestion
            <li in:fade|local={{ duration: 100 }} out:fade|local={{ duration: 100 }}>
```